.circleci/config.yml

```
@@ -95,18 +95,24 @@ jobs:
           name: Post Test Results to GitHub
           command: bundle exec danger || echo "danger failed"
           when: always  # Run this even when tests fail
-  'Execute tests on macOS (Xcode 11.0.0, Ruby 2.5)':
+  'Execute tests on macOS (Xcode 11.0, Ruby 2.5)':
     <<: *tests_macos
     macos:
-      xcode: '11.0.0'
+      xcode: '11.0'
     environment:
       _RUBY_VERSION: '2.5'
-  'Execute tests on macOS (Xcode 11.4.0, Ruby 2.6)':
+  'Execute tests on macOS (Xcode 11.7, Ruby 2.6)':
     <<: *tests_macos
     macos:
-      xcode: '11.4.0'
+      xcode: '11.7'
     environment:
       _RUBY_VERSION: '2.6'
+  'Execute tests on macOS (Xcode 12.3, Ruby 2.7)':
+    <<: *tests_macos
+    macos:
+      xcode: '12.3'
```

Since this PR failed with 12.3, this might need to be "12.3.0", I suppose...? 
https://discuss.circleci.com/t/xcode-12-3-release/385701

```suggestion
      xcode: '12.3.0'
```